desc/objectives/_coils.py

```diff
@@ -137,6 +161,31 @@ def _prune_coilset_tree(coilset):
             "Only use toroidal resolution for coil grids.",
         )
 
+        def broadcast_input(arr):
```

Comment:
It might make more sense to have this as a (maybe private) method of this objective, then when you make the target setter, you can reference this as well.